cli/cmd/configure_monitoring.go

```diff
@@ -40,8 +40,7 @@ var monitoringCmd = &cobra.Command{
 **Note:** If you are executing *keptn configure monitoring dynatrace*, the service flag is optional since Keptn automatically detects the services of a project. 
 See https://keptn.sh/docs/install/monitoring/ for more information.
 `,
-	Example: `keptn configure monitoring dynatrace --project=PROJECTNAME
-keptn configure monitoring prometheus --project=PROJECTNAME --service=SERVICENAME
```

Comment:
you are now missing the example for prometheus. Please, re-add this line

Reply:
Ok, I'll fix it.